example/package.json

```diff
@@ -12,29 +12,32 @@
     "test": "jest"
   },
   "dependencies": {
-    "@expo/vector-icons": "^10.2.0",
-    "@react-native-community/datetimepicker": "^2.4.0",
-    "@react-native-community/masked-view": "^0.1.10",
+    "@expo/vector-icons": "^10.0.0",
+    "@react-native-community/datetimepicker": "2.4.0",
+    "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/drawer": "^5.7.7",
     "@react-navigation/native": "^5.4.2",
-    "expo": "~37.0.3",
-    "expo-font": "^8.1.1",
-    "react": "~16.9.0",
-    "react-dom": "~16.9.0",
-    "react-native": "~0.61.5",
-    "react-native-gesture-handler": "^1.6.1",
-    "react-native-reanimated": "^1.9.0",
-    "react-native-safe-area-context": "^3.0.2",
-    "react-native-screens": "^2.8.0",
-    "react-native-svg": "^12.1.0",
-    "react-native-unimodules": "~0.9.0",
+    "@unimodules/core": "~5.3.0",
+    "@unimodules/react-native-adapter": "~5.4.0",
+    "expo": "^38.0.0",
+    "expo-file-system": "~9.0.1",
+    "expo-font": "~8.2.1",
+    "react": "16.11.0",
+    "react-dom": "16.11.0",
+    "react-native": "0.62.2",
```

Comment:
are these the right versions that expo looks for? im not sure sdk 38 supports 16.11 but maybe it does?

Reply:
Running app using yarn start runs smoothly with all components rendering and no errors so I guess it does support it. Have tested on Android & Web. Not on IOS though.